NN_dlaRadnych.js

```diff
@@ -127,6 +129,8 @@ nerthus.graf['panelTop']= 'http://cdn.rawgit.com/akrzyz/nerthusaddon/master/img/
 nerthus.graf['dazed']   = 'http://cdn.rawgit.com/akrzyz/nerthusaddon/master/img/dazed.png'
 nerthus.graf['loading'] = 'http://cdn.rawgit.com/akrzyz/nerthusaddon/master/img/loadingnerthus.png'
 nerthus.graf['chat']    = 'http://cdn.rawgit.com/akrzyz/nerthusaddon/master/img/chatPanel2.png'
+nerthus.graf['zodiac']  = 'http://cdn.rawgit.com/akrzyz/nerthusaddon/master/img/zodiacIcons.gif'
```

Comment:
w sumie to nie ma co tu dawać grafiki bo to robi zależność zodiaka do tego modułu, lepiej w samym zodiaku dodać jakieś pole np: nerthus.zodiac.icon = nerthus.addon.fileUrl("img/zodiacIcon.gif")